evals/skippy-waiting-prefix-ab.py

```
@@ -0,0 +1,476 @@
+        "config": str(config_path),
+        "log": str(log_path),
+        "requests": requests,
+        "summary": summarize(requests, events, makespan_ms),
+    }
+
+
+def aggregate(cells: list[dict[str, Any]]) -> list[dict[str, Any]]:
+    rows = []
+    for version in ("old", "new"):
+        summaries = [cell["summary"] for cell in cells if cell["version"] == version]
+        median = lambda key: statistics.median(float(row[key]) for row in summaries)
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `median` as a `def`

(E731)